### PR TITLE
Views: Pill sorting in Cards

### PR DESCRIPTION
**Pull request by gblakeman**
Adds some sorting functions to the primary card views that custom-sort the small pills based on content.

-----

Includes #104 

Relevant un-reviewed files:
```
src/Presenter.ts
src/CardPrimary.svelte
src/CardsLayout.svelte
```

### CODE REVIEW

File: src/Presenter.ts [hunk 1]
```diff
@@ -153,10 +170,12 @@ const setEntPstItem = (
           imgType: 'round' | 'square' | 'rect',
         } = null;
 
-        // tktk - exlude some things for right now
+        // tktk - exlude some things for now
```
Comment (gblakeman) [inline src/Presenter.ts:173]: New, down to `:178` (not in #104)

File: src/views/CardsLayout.svelte [hunk 2]
```diff
@@ -23,9 +23,22 @@
   const isDisabled = (
     itemsLocal: Array<RealishViewListItem>,
     isTrayOpenLocal: boolean,
-  ) => (
+  ): boolean => (
     !(itemsLocal && itemsLocal.length) || isTrayOpenLocal);
 
+  const isRemixDisabled = (
```
Comment (gblakeman) [inline src/views/CardsLayout.svelte:29]: Block already reviewed in #104

File: src/Presenter.ts [hunk 3]
```diff
@@ -130,12 +130,29 @@ const setEntPstItem = (
       imgType: 'round' | 'square' | 'rect',
     } = Data.imageContent(record.baseEntity.id, record.baseEntity.value);
 
+    // assemble list for dropdown
```
Comment (gblakeman) [inline src/Presenter.ts:133]: This was already reviewed in #104

File: src/Tools.ts [hunk 4]
```diff
@@ -41,6 +41,37 @@ const addOrRemoveItemId = (
...
+ * source [link]{@link https://svelte.dev/repl/0ace7a508bd843b798ae599940a91783?version=3.16.7}.
+ *
+ * @kind function
+ * @name clickOutsideSvelte
+ *
+ * @param {Object} element The `HTMLElement` to watch for clicks _outside_ of.
+ *
+ * @returns {Function} Destroys the listener after use.
+ */
+const clickOutsideSvelte = (element: HTMLElement) => {
```
Comment (lynnerang) [inline src/Tools.ts:57]: I know I saw this already, but this seems really nice.  Might try to find a way to use it in Specter at some point

Comment (gblakeman) [reply src/Tools.ts:57]: Nice, I’m using it with two different components and it works well. Happy to add it to the shared package so we can refine it together.

**Review by lynnerang — APPROVED** [review]
👍 🙌

**Pull request merged**